src/tools/importer/README.md

```diff
@@ -1,33 +1,161 @@
+
+### Import Data
 
-#### Config reference:
+To import data, run:
 
-Property Name  | Abbreviation |  Description
--------------- | ------------ | -------------
-address        | a            | thrift service address.
-batch          | b            | batch insert size.
-column         | c            | vertex and edge's column.
-file           | f            | data file path.
-help           | h            | help message.
-ranking        | k            | the edge have ranking data.
-schema         | m            | specify the schema name.
-name           | n            | specify the space name.
-timeout        | o            | thrift connection timeout.
-pswd           | p            | thrift service password.
-retry          | r            | thrift connection retry number.
-stat           | s            | print statistics info.
-type           | t            | data type. vertex or edge.
-user           | u            | thrift service username.
+```
+> java -jar importer-1.0.0-beta.jar <OPTIONS>
+```
+
+|Property Name  | Abbreviation |  Description| Example |
+|:----|:----:|:----:|:----:|
+|--address        | -a            | graphd service address.| 127.0.0.1:3699 |
+|--batch          | -b            | batch insert size.|2|
+|--name           | -n            | specify the space name.| myspace_test2 |
+|--schema         | -m            | specify the tag or edgetype name.| student |
+|--column         | -c            | properties of tag or edge to be inserted, separated by a comma | name,age |
+|--ranking        | -k            | whether the edge has ranking value. Default is false| true/false|
+|--file           | -f            | data file| ./tmp/data.txt |
+|--errorPath    | -d            | error log file | ./tmp/error.log |
+|--help           | -h            | list help|
+|--timeout        | -o            | specify connection timeout, in millisecond| 3000 |
+|--pswd           | -p            | graphd service password||
+|--connectionRetry       | -r            | connection retry times|
+|--executionRetry       | -e           | thrift execution retry times|
+|--stat           | -s            | print statistics info| true/false |
+|--type           | -t            | indicate to insert vertex properties or edge properties| vertext/edge|
+|--user           | -u            | graphd service username|
+
+> For edge type, if 'ranking' column is specified, '--ranking' option should be set to `true`.
+
+### Example
+
+The examples are based on [Insert Data Section of get-started.md](../../../docs/get-started.md#insert-data).
+
+
+E1. Insert vertices 200, 201, 202.
+
+data.csv
+
+```
+200,"Monica",16,"female"
+201,"Mike",18,"male"
+202,"Jane",17,"female"
+```
+
+Example command:
+
+```
+> java -jar importer-${VERSION}.jar --address 127.0.0.1:3699 --name myspace_test2 --schema student -u user -p password -t vertex --file data.txt --column name,age,gender --batch 2
+```
+
+E2. Insert `select` edges.
```

Comment:
insert edges selected or `select` is an edge type?  I'm confusing. I think 
**Insert selected edges will be better.**

Reply:
In getStarted.md, there is an edge type named `select`